components/script/dom/bluetoothremotegattserver.rs

```diff
@@ -127,4 +138,53 @@ impl BluetoothRemoteGATTServerMethods for BluetoothRemoteGATTServer {
             _ => unreachable!()
         }
     }
+
+    // http://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattserver-getprimaryservices
+    fn GetPrimaryServices(&self, service: Option<StringOrUnsignedLong>)
```

Comment:
Same proposes like at GetPrimaryServices.
Note: We can return personal error types like 'return Err(Type("New error message".to_owned()))' for this use the use 'dom::bindings::error::Error::Type';